style.css

```diff
@@ -0,0 +1,315 @@
+}
+
+.status {
+  size: px;
+}
+
+.header-items {
+  width: 375px;
+  height: 20px;
+  position: absolute;
+  padding-top: 20px;
+}
+
+.my-logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: absolute;
+  width: 105px;
+  height: 36px;
+  left: 12px;
+  top: calc(50% - 36px / 2);
+}
+
+.logo-text {
+  height: 16px;
+  font-family: 'Roboto';
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 16px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  white-space: nowrap;
+}
+
+.menu-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 66px;
+  height: 66px;
+  right: 24px;
+  top: calc(50% - 66px / 2);
+  border-radius: 8px;
+}
+
+.menu-btn-img {
+  background: none;
+  border: none;
+}
+
+.headline-inner {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.primary-text {
+  color: #7d73fc;
+  font-size: 48px;
+  font-weight: 700;
+  font-family: 'Roboto';
+  margin-bottom: -10px;
+  margin-top: 80px;
+}
+
+.supporting-text {
+  color: #edecff;
+  font-size: 32px;
+  font-weight: 500;
+  font-family: 'Roboto';
+}
+
+.headline {
+  margin-top: 0;
+  padding-top: 0;
+  height: 70%;
+  background-image: url("images/circles.png");
+  background-repeat: no-repeat;
+}
+
+.text-container {
+  margin: 10px;
+  padding: 2px;
+  height: 100%;
+}
+
+h1 {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 600;
+  font-size: 55px;
+  padding-top: 0;
+  margin-top: 5px;
+}
+
+.p-section {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 15px;
+}
+
+p {
+  width: 327px;
+  height: 170px;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #d8d8d4;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding-left: 10px;
+}
+
+.support {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 24px;
+  padding: 0;
+}
+
+.frame {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  gap: 24px;
+  width: 303px;
+  height: 214px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  font-family: 'Roboto',;
+  font-weight: 100;
+}
+
+.language {
+  width: auto;
+  height: auto;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  display: flex;
+  align-items: center;
+  color: #f5f5ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 0;
+  line-height: 38px;
+}
+
+.framework {
+  width: auto;
+  height: auto;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  display: flex;
+  align-items: center;
+  color: #f5f5ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 20px;
+  line-height: 38px;
+}
```

Comment:
- [ ] Please, 🙏🏽 correct the right margin on Framework to look similar to the Figma template.

- Figma template

![image](https://user-images.githubusercontent.com/58642949/185739068-d804250a-a6bd-4d0b-8f06-ab4cea393a03.png)

- Yours

![image](https://user-images.githubusercontent.com/58642949/185739075-11311e09-9462-40b4-9dbd-943d2a2f6204.png)